XamlControlsGallery/ControlPages/ListViewPage.xaml

```diff
@@ -109,99 +171,324 @@
+            <StackPanel>
+                <TextBlock TextWrapping="Wrap" Margin="0 0 0 15">
+                    Switch the toggle on the right to enable sticky group headers, which makes the headers stay put at the top of the ListView while scrolling.
+                </TextBlock>
+                <ListView 
+                x:Name="GroupedListViewCtrl"
                 ItemsSource="{x:Bind ContactsCVS.View, Mode=OneWay}"
                 ItemTemplate="{StaticResource ContactListViewTemplate}"
                 SelectionMode="Single"
                 ShowsScrollingPlaceholders="True"
                 Height="400"
-                MinWidth="550"
+                Width="400"
                 BorderThickness="1"
-                BorderBrush="DimGray">
-                <ListView.ItemsPanel>
-                    <ItemsPanelTemplate>
-                        <ItemsStackPanel AreStickyGroupHeadersEnabled="False" />
-                    </ItemsPanelTemplate>
-                </ListView.ItemsPanel>
-
-                <ListView.GroupStyle>
-                    <GroupStyle >
-                        <GroupStyle.HeaderTemplate>
-                            <DataTemplate x:DataType="local1:GroupInfoList">
-                                <TextBlock Text="{x:Bind Key}"
-                                       Style="{ThemeResource TitleTextBlockStyle}"/>
-                            </DataTemplate>
-                        </GroupStyle.HeaderTemplate>
-                    </GroupStyle>
-                </ListView.GroupStyle>
-            </ListView>
+                BorderBrush="DimGray"
+                HorizontalAlignment="Left">
+                    <ListView.ItemsPanel>
+                        <ItemsPanelTemplate>
+                            <ItemsStackPanel x:Name="GroupedStackPanel" AreStickyGroupHeadersEnabled="False" Loaded="StackPanel_loaded" />
+                        </ItemsPanelTemplate>
+                    </ListView.ItemsPanel>
 
+                    <ListView.GroupStyle>
+                        <GroupStyle >
+                            <GroupStyle.HeaderTemplate>
+                                <DataTemplate x:DataType="local1:GroupInfoList">
+                                    <Border AutomationProperties.Name="{x:Bind Key}">
+                                        <TextBlock Text="{x:Bind Key}" Style="{ThemeResource TitleTextBlockStyle}"/>
+                                    </Border>
+                                </DataTemplate>
+                            </GroupStyle.HeaderTemplate>
+                        </GroupStyle>
+                    </ListView.GroupStyle>
+                </ListView>
+            </StackPanel>
             
+
+
             <local:ControlExample.Xaml>
                 <x:String xml:space="preserve">
+&lt;!-- ListViews with grouped items must be bound to a CollectionViewSource, as shown below. --&gt;
 &lt;CollectionViewSource x:Name="ContactsCVS" IsSourceGrouped="True"/&gt;
```

Comment:
Would be great to see more code pertaining to this. If that feels too cumbersome, maybe some more verbal explanation?